### PR TITLE
Release 2022.9.0

### PR DESCRIPTION
**Pull request by charlesbluca**
Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by charlesbluca** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,41 +1,41 @@
...
 
 requirements:
   build:
-    - maven >=3.6.0
+    - {{ compiler('rust') }}
+    - setuptools-rust >=1.4.1
   host:
     - pip
-    - python >=3.8
+    - python
```
Comment (ayushdg) [inline recipe/meta.yaml:26]: might need to add a min pin back to 3.8

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,41 +1,41 @@
...
+    - setuptools-rust >=1.4.1
   run:
-    - dask >=2022.3.0,<=2022.8.0
+    - python
```
Comment (ayushdg) [inline recipe/meta.yaml]: same as above

**Comment by charlesbluca** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml (hunk 1 above)
Comment (charlesbluca) [reply recipe/meta.yaml:26]: IIRC, this causes issues in conjunction with the conda build config file (see https://github.com/dask-contrib/dask-sql/pull/766) - think the issue here was that I forgot to commit the config specifying what Python versions to build for

Comment (charlesbluca) [reply recipe/meta.yaml:26]: My bad, you were right - looks like conda-forge handles its build version matrix differently from the nightlies.

I've made the suggested modifications and re-rendered, so things should be working now

**Review by ayushdg — APPROVED** [review]

**Review by galipremsagar — APPROVED** [review]

**Pull request merged**